webapp/src/main/webapp/components/snackbar/ExoNewsNotificationAlerts.vue

```diff
@@ -58,6 +58,15 @@ export default {
         });
       }
     });
+    this.$root.$on('restricted-space', (spaceDisplayName) => {
```

Comment:
```suggestion
    this.$root.$on('restricted-news-activity-access', (spaceDisplayName) => {
```
the message to be displayed will not contain space name